mvpa2/algorithms/group_clusterthr.py

```diff
@@ -283,7 +287,7 @@ def featuresegment_producer(ncols):
             # same code as above, just in parallel with joblib's Parallel
             thrmap = np.hstack(
                 Parallel(n_jobs=self.params.n_proc,
-                         pre_dispatch=self.params.n_proc,
+                         #pre_dispatch=self.params.n_proc,
```

Comment:
because the default of pre_dispatch set to 3*n_proc should be a better option to avoid "starving" the cores. however it doesn't do much anything anyway in this case